packages/noya-designsystem/src/utils/touchable.ts

```diff
@@ -0,0 +1,276 @@
+import type {
+  TouchData,
+  GestureUpdateEvent,
+  GestureStateChangeEvent,
+  PanGestureChangeEventPayload,
+  TapGestureHandlerEventPayload,
+  PanGestureHandlerEventPayload,
+  LongPressGestureHandlerEventPayload,
+} from 'react-native-gesture-handler';
+
+import type { Point } from 'noya-geometry';
+
+/**
+ * Types used by manual gesture utilities
+ */
```

Comment:
Did we take these functions from something open source, or is it all custom? If we took it from somewhere we should add a license/link

Reply:
100% custom :)